cassandra-driver.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ['README.md']
   s.rdoc_options << '--title' << 'Datastax Ruby Driver' << '--main' << 'README.md' << '--line-numbers'
 
-  s.required_ruby_version = '>= 2.2.0'
+  s.required_ruby_version = '>= 2.3.0'
```

Comment:
sorted_set requires ruby 2.3. I've seen libraries not consider bumps in minimum ruby versions breaking changes when the dropped ruby version(s) is passed EOL so I'll defer here to the maintainers w.r.t versioning.